ssc/cmod_mhk_eqns.cpp

```diff
@@ -15,16 +14,16 @@ void me_array_cable_length(ssc_data_t data)
 
     double devices_per_row, device_spacing_in_row, number_rows, row_spacing, cable_system_overbuild, floating_array, export_cable_redundancy, water_depth, number_devices, distance_to_shore;
 
-    VT_GET_INPUT(vt, "devices_per_row", devices_per_row)
-    VT_GET_INPUT(vt, "device_spacing_in_row", device_spacing_in_row)
-    VT_GET_INPUT(vt, "number_rows", number_rows)
-    VT_GET_INPUT(vt, "row_spacing", row_spacing)                    
-	VT_GET_INPUT(vt, "cable_system_overbuild", cable_system_overbuild)
-	VT_GET_INPUT(vt, "floating_array", floating_array)
-	VT_GET_INPUT(vt, "export_cable_redundancy", export_cable_redundancy)
-	VT_GET_INPUT(vt, "water_depth", water_depth)
-	VT_GET_INPUT(vt, "number_devices", number_devices)
-	VT_GET_INPUT(vt, "distance_to_shore", distance_to_shore)
+    vt_get_double(vt, "devices_per_row", &devices_per_row);
+    vt_get_double(vt, "device_spacing_in_row", &device_spacing_in_row);
+    vt_get_double(vt, "number_rows", &number_rows);
+    vt_get_double(vt, "row_spacing", &row_spacing);
+    vt_get_double(vt, "cable_system_overbuild", &cable_system_overbuild);
+    vt_get_double(vt, "floating_array", &floating_array);
+    vt_get_double(vt, "export_cable_redundancy", &export_cable_redundancy);
+    vt_get_double(vt, "water_depth", &water_depth);
+    vt_get_double(vt, "number_devices", &number_devices);
+    vt_get_double(vt, "distance_to_shore", &distance_to_shore);
```

Comment:
Will get_double be more or less difficult to change if we go back to floats?

Reply:
we could have a `vt_get_float` and a `vt_get_long_double`

Reply:
but i see your point. vartab.h uses `ssc_number_t` so it would be better to have a vt_get_number instead to be consistent.

Reply:
fixed in 2317078bc48c4a63b20430b181ad4011ed2ff01b